CHANGELOG.md

```diff
@@ -14,7 +14,7 @@
 * **Faster ocr_only speed for partitioning PDF and images.** Use `unstructured_pytesseract.run_and_get_multiple_output` function to reduce the number of calls to `tesseract` by half when partitioning pdf or image with `tesseract`
 * **Adds data source properties to fsspec connectors** These properties (date_created, date_modified, version, source_url, record_locator) are written to element metadata during ingest, mapping elements to information about the document source from which they derive.
 * **Non-HTML text files now return unstructured-elements as opposed to HTML-elements.** Previously the text based files that went through `partition_html` would return HTML-elements but now we preserve the format from the input using `source_format` argument in the partition call.
-
+* **Ingest tests cleanup** Add cleanup to all ingest tests to delete created content by default. This remedies memory issues of running all the tests on a single runner in the CI.
```

Comment:
we should probably keep changelog notes specific to code changes / can omit notes that relate to tests